examples/speaker_recognition/spkr_get_emb.py

```diff
@@ -54,7 +54,6 @@ def main(cfg):
     model_path = os.path.join(log_dir, '..', 'spkr.nemo')
     speaker_model = ExtractSpeakerEmbeddingsModel.restore_from(model_path)
     speaker_model.setup_test_data(cfg.model.test_ds)
-
     trainer.test(speaker_model)
```

Comment:
Can you use the trainer.prepare_test() checks to ensure you dont eval on multiple gpus? Look at the speech_to_label.py script for usage.

Reply:
I never faced issues with multi-GPU for extraction of my embeddings, I use it to gather my output. Is it necessary?

Reply:
See above.